watertap/unit_models/ion_exchange_0D.py

```diff
@@ -141,60 +203,129 @@ class IonExchangeODData(InitializationMixin, UnitModelBlockData):
 
+    CONFIG.declare(
+        "isotherm",
+        ConfigValue(
+            default=IsothermType.langmuir,
+            domain=In(IsothermType),
+            description="Designates the isotherm type to use for equilibrium calculations",
+        ),
+    )
+
     def build(self):
         super().build()
 
-        ### REFERENCES ###
-        # LeVan, M. D., Carta, G., & Yon, C. M. (2019).
-        # Section 16: Adsorption and Ion Exchange.
-        # Perry's Chemical Engineers' Handbook, 9th Edition.
+        """
+        REFERENCES
+
+        LeVan, M. D., Carta, G., & Yon, C. M. (2019).
+        Section 16: Adsorption and Ion Exchange.
+        Perry's Chemical Engineers' Handbook, 9th Edition.
+
+        Crittenden, J. C., Trussell, R. R., Hand, D. W., Howe, K. J., & Tchobanoglous, G. (2012).
+        Chapter 16: Ion Exchange.
+        MWH's Water Treatment (pp. 1263-1334): John Wiley & Sons, Inc.
+
+        DOWEX Ion Exchange Resins Water Conditioning Manual
+        https://www.lenntech.com/Data-sheets/Dowex-Ion-Exchange-Resins-Water-Conditioning-Manual-L.pdf
+
+        Inamuddin, & Luqman, M. (2012).
+        Ion Exchange Technology I: Theory and Materials.
 
-        # Crittenden, J. C., Trussell, R. R., Hand, D. W., Howe, K. J., & Tchobanoglous, G. (2012).
-        # Chapter 16: Ion Exchange.
-        # MWH's Water Treatment (pp. 1263-1334): John Wiley & Sons, Inc.
+        Vassilis J. Inglezakis and Stavros G. Poulopoulos
+        Adsorption, Ion Exchange and Catalysis: Design of Operations and Environmental Applications (2006).
+        doi.org/10.1016/B978-0-444-52783-7.X5000-9
 
-        # DOWEX Ion Exchange Resins Water Conditioning Manual
-        # https://www.lenntech.com/Data-sheets/Dowex-Ion-Exchange-Resins-Water-Conditioning-Manual-L.pdf
+        Michaud, C.F. (2013)
+        Hydrodynamic Design, Part 8: Flow Through Ion Exchange Beds
+        Water Conditioning & Purification Magazine (WC&P)
+        https://wcponline.com/2013/08/06/hydrodynamic-design-part-8-flow-ion-exchange-beds/
 
-        # Inamuddin, & Luqman, M. (2012).
-        # Ion Exchange Technology I: Theory and Materials.
+        Clark, R. M. (1987). 
+        Evaluating the cost and performance of field-scale granular activated carbon systems. 
+        Environ Sci Technol, 21(6), 573-580. doi:10.1021/es00160a008
 
-        # Michaud, C.F. (2013)
-        # Hydrodynamic Design, Part 8: Flow Through Ion Exchange Beds
-        # Water Conditioning & Purification Magazine (WC&P)
-        # https://wcponline.com/2013/08/06/hydrodynamic-design-part-8-flow-ion-exchange-beds/
+        Croll, H. C., Adelman, M. J., Chow, S. J., Schwab, K. J., Capelle, R., Oppenheimer, J., & Jacangelo, J. G. (2023). 
+        Fundamental kinetic constants for breakthrough of per- and polyfluoroalkyl substances at varying empty bed contact times: 
+        Theoretical analysis and pilot scale demonstration. 
+        Chemical Engineering Journal, 464. doi:10.1016/j.cej.2023.142587
```

Comment:
This should go in a doc string towards beginning of the file describing the model.

Reply:
Moved to top of file.